lib/block_view.go

```diff
@@ -4265,7 +4265,7 @@ func (bav *UtxoView) ConnectBlock(
 		}
 		if isLastBlockInEpoch {
 			var utxoOperations []*UtxoOperation
-			utxoOperations, err = bav.RunEpochCompleteHook(blockHeight, blockHeader.ProposedInView, blockHeader.TstampNanoSecs)
+			utxoOperations, err = bav.RunEpochCompleteHook(blockHeight, blockHeader.GetView(), blockHeader.TstampNanoSecs)
```

Comment:
This currently defines a 0 view for PoW epochs. This can be cleaned up by the assumption that view = height for all PoW blocks and epochs. If the PoW -> PoS transition happens mid epoch, this change makes the leader index check completely safe and predictable